Assets/AirConsole/scripts/AirConsole.cs

```diff
@@ -1930,6 +1939,26 @@ public class NotReadyException: SystemException {
 #endif
 
 #endregion
+		
+#if UNITY_ANDROID || UNITY_EDITOR
+		/// <summary>
+		/// Enables player silencing
+		/// </summary>
+		private void EnablePlayerSilencing () {
+			if (!IsAirConsoleUnityPluginReady ()) {
+				throw new NotReadyException ();
+			}
+			JObject msg = new JObject {
+				{ "action", "enablePlayerSilencing" }
+			};
+
+			AllocateDeviceSlots (0);
```

Comment:
Where does that function live, I can't find it?

Reply:
What function are you referring to?
enablePlayerSilencing would be in api 1.9.0-alpha (branch of the same name in api) and as message in the js files you commented on before

Reply:
Sorry I meant the "AllocateDeviceSlots"

Reply:
That is a function in AirConsole.cs, AirConsole class